tests/libvmops/run.go

```diff
@@ -56,20 +58,19 @@ func RunVMAndExpectLaunchWithRunStrategy(virtClient kubecli.KubevirtClient, vm *
 	return vm
 }
 
-func updateVMRunningStatus(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine, runStrategy *v1.VirtualMachineRunStrategy) (*v1.VirtualMachine, error) {
-	var err error
-	var updatedVM *v1.VirtualMachine
-
-	EventuallyWithOffset(1, func() error {
-		updatedVM, err = virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
-		ExpectWithOffset(1, err).ToNot(HaveOccurred())
-		updatedVM.Spec.Running = nil
-		updatedVM.Spec.RunStrategy = runStrategy
-		updatedVM, err = virtClient.VirtualMachine(vm.Namespace).Update(context.Background(), updatedVM, metav1.UpdateOptions{})
-		return err
-	}, 30*time.Second, 1*time.Second).Should(Succeed())
-
-	return updatedVM, err
+func updateVMRunningStatus(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine, runStrategy v1.VirtualMachineRunStrategy) (*v1.VirtualMachine, error) {
+	// Prepare the patch data
```

Comment:
No need to express in a comment what is obvious, it adds no useful information.

Reply:
ack